sentry-android-integration-tests/test-app-sentry/build.gradle.kts

```diff
@@ -0,0 +1,53 @@
+plugins {
+    id("com.android.application")
+}
+
+android {
```

Comment:
This sample adds Sentry Android with NDK support, but its a fat APK with all the ABIs.
What do you think if we compare the size when compiling a single ABI?
https://developer.android.com/studio/build/configure-apk-splits
That's what most people are or should be doing anyway.

Reply:
This has the same effect in practice (I've checked the sizes locally): https://github.com/getsentry/sentry-java/pull/2171/commits/0e00c1a2de1083a7f83078357eea79b30493d984 - you fine with that?